.github/workflows/ci.yml

```diff
@@ -26,19 +26,20 @@ jobs:
           - ubuntu-20.04
         tool:
           # Note: Specifying the version of valgrind and wasm-pack is not supported.
-          - cargo-hack,cargo-llvm-cov,cargo-minimal-versions,parse-changelog,cross,nextest,shellcheck,shfmt,valgrind,wasm-pack,wasmtime,mdbook
-          - cargo-hack@0.5.14,cargo-llvm-cov@0.4.5,cargo-minimal-versions@0.1.4,parse-changelog@0.4.8,cross@0.2.1,nextest@0.9.11,shellcheck@0.8.0,shfmt@3.5.0,wasmtime@0.37.0,mdbook@0.4.18
+          # minifier is supported by cargo-binstall (through quickinstall)
```

Comment:
If possible, I would like to choose a crate maintained by the cargo-binstall maintainer or collaborator for testing. (maybe cargo-watch?)
This is because there is little risk of support for cargo-binstall being removed.